application/core/network/library/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
WAT

Reply:
Seems, it came from marvel-project. Also, that unit-tests has a lot doubtful tools, including Roboelectric...
I think, we don't need all that for our sample